pkg/cri/store/stats/stats.go

```diff
@@ -16,12 +16,56 @@
 
 package stats
 
-import "time"
-
 // ContainerStats contains the information about container stats.
 type ContainerStats struct {
-	// Timestamp of when stats were collected
-	Timestamp time.Time
+	ContainerCPUStats
+	ContainerMemoryStats
+	ContainerFileSystemStats
+}
+
+type ContainerCPUStats struct {
+	// Timestamp in nanoseconds at which the information were collected. Must be > 0.
+	Timestamp int64
```

Comment:
CRI [fields](https://github.com/kubernetes/cri-api/blob/master/pkg/apis/runtime/v1/api.proto#L1542) use int64, figured it would be easier than converting

Reply:
Since this is an internal struct that needs to be converted on the way out anyway, I think it'd make more sense to centralize the conversion and maintain the standard `time.Time` here.